common/djangoapps/student/forms.py

```diff
@@ -242,6 +243,24 @@ def validate_name(name):
         raise forms.ValidationError(_('Full Name cannot contain the following characters: < >'))
 
 
+def validate_org_size(org_size):
```

Comment:
In same form we have used `clean_field_name` for some fields and using `validators` for other fields. Don't you think, we should use either `validators` or `clean_field_name` methods. Let's have a discussion which is better for validation in our case.

https://docs.djangoproject.com/en/1.11/ref/forms/validation/#form-and-field-validation

Reply:
I have moved the org related validation in `clean_org_name` method since it makes the most sense there.

Reply:
But clean method is more appropriate place for this scenario. as we are accessing more then 1 field.